FEATURES.md

```diff
@@ -4,27 +4,18 @@
 
 ## Features
 
-## v2.0.0.beta1 - 2021-07-20
-
-- Generate new apps using `app/` directory
-
-## v2.0.0.alpha8 - 2021-05-19
-
-## v2.0.0.alpha7.1 - 2021-03-09
-
-## v2.0.0.alpha7 - 2021-03-08
-
-## v2.0.0.alpha6 - 2021-02-10
-
-## v2.0.0.alpha5 - 2021-01-12
-
-## v2.0.0.alpha4 - 2021-12-07
-
-## v2.0.0.alpha3 - 2021-11-09
-
-## v2.0.0.alpha2 - 2021-05-04
-
-## v2.0.0.alpha1 - 2019-01-30
+## v2.0.0 - 2021-11-22
+
+- App class built around a container for managing app components; code from app/ is loaded into the app
```

Comment:
@timriley As first item, please write something about `app/` centrism

Reply:
@jodosha as in, the app is always there, but the slices can be opt-in only if you want them?

Reply:
@jodosha maybe you can have a go at writing this, since I think you have a pretty clear idea in your mind about what to write?